render/detailed/connections.go

```diff
@@ -145,7 +145,7 @@ func outgoingConnectionsSummary(topologyID string, r report.Report, n report.Nod
 				if !ok {
 					continue
 				}
-				key := newConnection(n, node, port, remoteEndpointID, localAddr)
```

Comment:
Discussed in person. Of course the inbound internet node *has* no inbound connections. It originates all connections, so it only *has* outbound connections.